WTG.Analyzers.TestFramework/TestData/SampleDataSet.cs

```diff
@@ -49,6 +49,18 @@ where name.StartsWith(prefix, StringComparison.Ordinal)
 			return tmp.ToImmutableArray();
 		}
 
+		static int FileNameStartIndex(string prefix, string name)
+		{
+			var index = name.LastIndexOf('.', name.Length - 1, name.Length - prefix.Length);
+
+			if (index < 0)
```

Comment:
What if `index == 0`?

Reply:
Then that would indicate that `name` starts with a '.', but this can't happen in practice because `prefix` won't be empty and so the range we check won't include index 0.